icon_font/_icon-font.scss

```diff
@@ -0,0 +1,79 @@
+@font-face {
+font-family: 'icomoon';
+src: url('/icon_font/icomoon.woff2') format('woff2');
+src: url('/icon_font/icomoon.woff') format('woff');
+font-weight: normal;
+font-style: normal;
+font-display: block;
```

Comment:
Please, add indents.
Check all the files.

Reply:
indents added

Reply:
> Please, add indents.
> Check all the files.
indents added